### DISCUSSION TITLE
I'm not sure if I have a bug or a user error, but installing prebuilt docker issue

### DISCUSSION THREAD

**Discussion by keyosuke** [JeffreyCA/spleeter-web#113] [General]
I'm using Anaconda console on Windows 11 with Docker.  Prebuilt containers and self-built, the same result.

I blew way my old docker containers for Spleeter Web and renamed the directory it was installed into, for a fresh start.  I then following the instructions to set it up, as I've done in the past, and the containers repopulated into Docker Dashboard, but when I try to log into the page at 127.0.0.1:8000 I get nothing.  Some attempts at installing it lead to docker containers seeming to get in a restarting loop, but I don't see an issue with the latest install, other than that it's not accepting connections on 8000.     I have my old spleeter-web directory, so is there any way I can roll back to it or troubleshoot?

**Comment by JeffreyCA** [comment]
Hi, can you share some logs of any errors you see?

Can you also check if port 80 works? I did recently update the README
instructions so that the Docker app runs on port 80 through nginx instead
of 8000.

Yes you can checkout an older git commit and self-build the Docker
containers.

**Reply by keyosuke** [reply]
That might explain a lot of what I'm seeing, the port 80 think.  I don't see any errors at the moment, but is there a log I can dump or collect?  I'll try port 80 now.

**Reply by keyosuke** [reply]
It looks like I have a working instance on portr 80.  I had many failed attempts at this that did have errors, but clearly that was resolved, even if I couldn't tell.  Thank you for your help.  I thought I read it all, but I must have missed the change.

**Reply by keyosuke** [reply]
Time for me to hit the documentation to learn how to change that port.  My external router blocks 80 and I cannot change that, because my internet service provider hosts a remote login page on it and I have no access to move or stop their hosting of that.  Kind of a shame.  I remember seeing that you can move the port in the docs, though.

**Reply by keyosuke** [reply]
I tried setting NGINX_PORT=8000 in .env file and restarted in docter and it went into a restart loop with the following error.  

spleeter-web_celery-slow_1
jeffreyca/spleeter-web-backend:latest
RUNNING


  File "/webapp/api/tasks.py", line 15, in <module>

    from .separators.demucs_separator import DemucsSeparator

  File "/webapp/api/separators/demucs_separator.py", line 7, in <module>

    from demucs.pretrained import load_pretrained, PRETRAINED_MODELS

ImportError: cannot import name 'load_pretrained' from 'demucs.pretrained' (/usr/local/lib/python3.7/site-packages/demucs/pretrained.py)

Starting Celery (slow)

**Comment by JeffreyCA** [comment]
Are you building the Docker containers from source? It looks like you're on
an older commit but using a running a newer Demucs dependency which isn't
compatible. You can also run "docker-compose pull" to get the latest
prebuilt containers.

**Reply by keyosuke** [reply]
This fixed my issue with Celerys restart looping.
